src/test/ui/lint/issue-80988.rs

```diff
@@ -0,0 +1,16 @@
+// Regression test for #80988
+//
+// check-pass
+
+#![forbid(warnings)]
+
+#[deny(warnings)]
+//~^ WARNING incompatible with previous forbid
+//~| WARNING being phased out
+//~| WARNING incompatible with previous forbid
+//~| WARNING being phased out
+//~| WARNING incompatible with previous forbid
+//~| WARNING being phased out
+//~| WARNING incompatible with previous forbid
```

Comment:
sigh. I'm sure the 4x repetition is an artifact of the existing code, and we don't want to deal with fixing it on something that we are beta-backporting. (And I hope that the compiler's de-duplication of diagnostics turns it into a single message.)

But still: Sigh.